content/en/building/tasks/_index.md

```diff
@@ -5,4 +5,62 @@ identifier: "Tasks"
 weight: 3
 description: >
   Overview and configuration of CHT Tasks
+aliases:
+   - /building/features/tasks/
+   - /apps/features/tasks/
 ---
+
+Tasks help CHWs plan their day by prompting them to complete follow-up visits and other activities. The list might include upcoming scheduled ANC or Immunization visits, treatment or referral follow-ups, or other required activities such as a household survey.
+
+{{< figure src="tasks-mobile.png" link="tasks-mobile.png" class="left col-3 col-lg-3" >}}
+
+{{< figure src="tasks-desktop.png" link="tasks-desktop.png" class="left col-9 col-lg-9" >}}
+
+
+## Main List
+
+{{< figure src="tasks-mobile.png" link="tasks-mobile.png" class="right col-6 col-lg-3" >}}
+
+On the Tasks tab is a consolidated list of tasks for all people and families that the user looks after. The task definition determines how long the task will show on this list before and after it is due.
+
+Each task has an icon on the left side which indicates which type of task it is. The first bold line of text is the person or family that the task is about. The second line of text is the title of the task. The due date for the task is located in the upper right-hand corner. If a task is overdue, the due date will be red.
+
+Tasks are listed in order of due date. Tasks that are past due will appear at the top of the list. CHWs should strive to complete tasks before they are overdue. Many programs add targets to track task completion and timeliness.
+
+
+<br clear="all">
+
+## Care Guides
+
+When a CHW clicks on a task, the care guide configured for that task displays. CHWs are then guided through questions for that specific workflow.
+
+<div class="container">
+  <div class="row">
+{{< figure src="tasks-care1.png" link="tasks-care1.png" class="col-6 col-lg-3" >}}
+{{< figure src="tasks-care2.png" link="tasks-care2.png" class="col-6 col-lg-3" >}}
+{{< figure src="tasks-care3.png" link="tasks-care3.png" class="col-6 col-lg-3" >}}
+{{< figure src="tasks-care4.png" link="tasks-care4.png" class="col-6 col-lg-3" >}}
+  </div>
+</div>
+
+For more information on Care Guides, see the “[Decision Support for Care Guides]()” section of this overview. 
```

Comment:
The link is missing for the `Decision Support for Care Guides`

Reply:
I'll fix this issue [here](https://github.com/medic/cht-docs/issues/1664)